README.md

```diff
@@ -53,8 +53,33 @@ To change your local development app network, update the network key for
 42 - Kovan
 137 - Polygon
 42161 - Arbitrum
+31337 - Local test node with forked mainnet (run in separate process)
 ```
 
+## Local test node
+
+For local development purposes you may want to use a test node with a forked mainnet state.
+
+If you keep you RPC_URL in local .env
+```
+export RPC_URL=$(grep ALCHEMY_URL .env | cut -d '=' -f2)
+```
+
+Hardhat:
+```
+npm run node
```

Comment:
Maybe add a note that this command should be run from `balancer-sdk`.

Also listing available network ids on the readme could be helpful.